tests/lifecycle/tests/lifecycle_affinity_required_pods.go

```diff
@@ -11,43 +11,69 @@ import (
 	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/config"
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
-	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/nodes"
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/pod"
 
 	tshelper "github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/helper"
 	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/parameters"
 )
 
-var _ = Describe("lifecycle-affinity-required-pods", Serial, func() {
+var _ = Describe("lifecycle-affinity-required-pods", func() {
+	var randomNamespace string
+	var origReportDir string
+	var origTnfConfigDir string
 
 	configSuite, err := config.NewConfig()
 	if err != nil {
 		glog.Fatal(fmt.Errorf("can not load config file: %w", err))
 	}
 
 	BeforeEach(func() {
-		err := tshelper.WaitUntilClusterIsStable()
-		Expect(err).ToNot(HaveOccurred())
+		randomNamespace = tsparams.LifecycleNamespace + "-" + globalhelper.GenerateRandomString(10)
 
-		By("Clean namespace before each test")
-		err = namespaces.Clean(tsparams.LifecycleNamespace, globalhelper.GetAPIClient())
+		By(fmt.Sprintf("Create %s namespace", randomNamespace))
+		err := namespaces.Create(randomNamespace, globalhelper.GetAPIClient())
 		Expect(err).ToNot(HaveOccurred())
 
-		By("Ensure all nodes are labeled with 'worker-cnf' label")
-		err = nodes.EnsureAllNodesAreLabeled(globalhelper.GetAPIClient().CoreV1Interface, configSuite.General.CnfNodeLabel)
+		By("Override default report directory")
```

Comment:
same comment as for https://github.com/test-network-function/cnfcert-tests-verification/pull/446. create  common functions for BeforeEach and AfterEach